examples/specs/window_transform.vl.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+    "description": "Bar graph showing the row number for each of the cells",
+    "width": 300,
+    "height": 50,
+    "data": {
+        "values": [
+            { "x": 1, "y": 28 }, { "x": 2, "y": 55 },
```

Comment:
Why do the data need "x" here at all?

Also, maybe rename this file to `window_row_number`?